src/core/tritonserver.h

```diff
@@ -633,6 +633,19 @@ TRITONSERVER_InferenceRequestSetRequestedOutputClassificationCount(
     TRITONSERVER_InferenceRequest* inference_request, const char* name,
     uint32_t count);
 
+/// Get the classification count of the requested output.
+/// \param inference_request The request object.
+/// \param name The name of the output.
+/// \param count Indicates how many classification values will be
+/// returned for the output. The 'count' highest priority values are
+/// returned. A value of 0 indicates that the output tensor
+/// will not be returned as a classification.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_EXPORT TRITONSERVER_Error*
+TRITONSERVER_InferenceRequestRequestedOutputClassificationCount(
```

Comment:
Why is this needed? The output shape indicates how many classifications there are.